source/utilities.tex

```diff
@@ -9439,9 +9439,8 @@
 \end{itemdecl}
 
 \begin{itemdescr}
-\pnum\effects  Equivalent to \tcode{shared_ptr(r).swap(*this)}.
-
-\pnum\returns  \tcode{*this}.
+\pnum\effects  Equivalent to:
+\tcode{shared_ptr(r).swap(*this); return *this;}
 
```

Comment:
We lost the \returns here?

Reply:
No. Look at the end of line 9443; there's a second statement, which happens to be a return statement.

Reply:
Oh eek. Can we make that a `codeblock` then?

Reply:
Or keep the \returns, maybe?

Reply:
Yes. I'm not keen on getting overly creative in compressing stuff. Readability is important. The `return *this = that;` is borderline, too (but I was going to let it slide).

Reply:
I've undone the changes in this area.